nixos/modules/services/networking/consul.nix

```diff
@@ -40,15 +29,6 @@ in
         '';
       };
 
-
-      webUi = mkOption {
```

Comment:
I believe `nixos/tests/consul.nix` needs to be adjusted to still evaluate given this change.